usr/share/rear/conf/default.conf

```diff
@@ -2414,10 +2414,21 @@ AUTOEXCLUDE_MULTIPATH=y
 # Automatically exclude automounter paths from the backup
 AUTOEXCLUDE_AUTOFS=
 
-# Automatically exclude filesystems mounted under directories given here
-# The default is /media to exclude USB devices mounted there.
+# Automatically exclude filesystems with mountpoints that are sub-directories
+# of the directories that are specified in the AUTOEXCLUDE_PATH array.
 # This is different from EXCLUDE_MOUNTPOINTS, which accepts only mountpoints.
-AUTOEXCLUDE_PATH=( /media )
+# Filesystems with mountpoints in the AUTOEXCLUDE_PATH array are not excluded.
+# For details see the layout/save/default/320_autoexclude.sh script.
+# For example a separated filesystem with mountpoint /tmp is not excluded
+# but the files in the /tmp directory are excluded from the backup
+# when an internal backup method is used by BACKUP_PROG_EXCLUDE=( '/tmp/*' ... )
+# so that this filesystem will be recreated as an empty filesystem.
```

Comment:
Hi @jsmeix I don't get it, why to mention this in the description of `AUTOEXCLUDE_PATH`? The comment above is not really related to `AUTOEXCLUDE_PATH`, it describes the behavior of `BACKUP_PROG_EXCLUDE`.
(I am asking because it is confusing, without a very careful reading one is left with the expectation that `/tmp` in `AUTOEXCLUDE_PATH` leads to adding '/tmp/*' to `BACKUP_PROG_EXCLUDE`.)

Reply:
I describe the default behaviour.
In default.conf there is (excerpts)
```
BACKUP_PROG_EXCLUDE=( '/tmp/*' ... )
...
AUTOEXCLUDE_PATH=( ... /tmp )
```
and both default settings together result that
for a separated filesystem with mountpoint /tmp
the files in the /tmp directory are excluded from the backup
(provided an internal backup method is used)
and this filesystem will be recreated as an empty filesystem
(with mountpoint /tmp).

In general regarding the include/exclude stuff in ReaR
cf. my comments like
https://github.com/rear/rear/issues/2229#issuecomment-531264805
and
https://github.com/rear/rear/issues/2772#issuecomment-1069119836

Reply:
Via
https://github.com/rear/rear/commit/81af5218f73aa3a9562340cb67b2df08428b2aed
I made the description in default.conf (hopefully) more clear
how the defaults for AUTOEXCLUDE_PATH and BACKUP_PROG_EXCLUDE
interact when /tmp is a separated filesystem.

Reply:
@jsmeix I understand that you are describing the default behavior, and the description is correct.

My problem is that the description occurs in a phrase starting "For example ...", which thus has the point of illustrating the behavior of `AUTOEXCLUDE_PATH` via an example. But the treatment of `BACKUP_PROG_EXCLUDE` is not relevant to the explanation of `AUTOEXCLUDE_PATH` - if one adds something else to `AUTOEXCLUDE_PATH`, it will not get added into `BACKUP_PROG_EXCLUDE`, unless the user adds it explicitly there. Actually, the `/tmp` entry in `AUTOEXCLUDE_PATH` is unrelated to the `/tmp/*` entry in `BACKUP_PROG_EXCLUDE` (except that both are somehow related to the /tmp directory), because the former excludes filesystems mounted under /tmp, while the latter excludes the content of the /tmp filesystem itself.
I believe that we need to decide whether this phrase is
- intended to explain the behavior of `AUTOEXCLUDE_PATH`, and then we should choose another example than /tmp - it is a poor example, because it gets treated specially by the defaults in a way that no other directory does - and a special case is never a good illustration of a general rule, or,
- intended as a complete description of the treatment of `/tmp` via the various defaults, but then the "For example" should be deleted.

Reply:
Or we can split the phrase in two and do both of course. I can send a PR once we agree on the approach.

Reply:
I used "for example" because it is
an example that /tmp is a separated filesystem
(normally /tmp is a directory of the root filesystem).
I used '/tmp' as an example because that was
what I had explicitly tested in
https://github.com/rear/rear/pull/2261
so I knew how the include/exclude stuff in ReaR
actually behaves - at least for me.
I won't dare to make generic statements how
that "hairy" include/exclude stuff in ReaR
behaves in general ;-)

@pcahyna 
of course feel free to make a PR with a better text.
I would much appreciate it.
In particular when you could also test how that
include/exclude stuff in ReaR behaves for you.
Perhaps you may find interesting/unexpected behaviour
with your tests where we may have to fix something
or at least better desribe how it actually works.

Reply:
#3131